packages/games/emulators/dosbox-sdl2/config/games/_Scan DOSBox Games.sh

```diff
@@ -12,20 +14,26 @@ rp_registerAllModules
 joy2keyStart
 ear >/dev/console
 
+#rm "/storage/.config/dosbox/games/*.conf"
+
 function create_launcher() {
     launcher_name="$1 ($2)"
 
-    dos_dirname="${1:0:6}~1"
-
-    mkdir -p "/storage/.config/dosbox/games/$launcher_name"
-    cp /storage/.config/dosbox/dosbox-SDL2.conf "/storage/.config/dosbox/games/$launcher_name"
-    cat <<EOF >>/storage/.config/dosbox/games/$launcher_name/dosbox-SDL2.conf
-mount c /storage/roms/pc/$(basename $data_dir)
-c:
-$2
-exit
-EOF
```

Comment:
This cat was not working for me, so I reverted it to the various echoes and I even edited them.